docs/user-guides/migrating-from-kube-lego.md

```diff
@@ -86,8 +86,10 @@ more relevant to cert-manager. For the rest of this guide, we'll assume you
 chose `letsencrypt-private-key`.
 
 Once done, we need to create this new resource in the `kube-system` namespace.
-This is where cert-manager looks for supporting resources for ClusterIssuers
-by default:
+By default, cert-manager stores supporting resources for ClusterIssuer's in the
```

Comment:
`ClusterIssuer`s? ClusterIssuers? I'm not sure what the correct plural is, they all look wrong!

Reply:
Fortunately we have a structured yaml file specifying how to pluralize it :)

https://github.com/jetstack/cert-manager/blob/52b555ef9ce6a2f21ac21c8d6dae4585a1af123c/contrib/charts/cert-manager/templates/clusterissuer-crd.yaml#L11-L12

I think using `ClusterIssuers` is best here.